iceoryx2/src/config.rs

```diff
@@ -535,7 +525,34 @@ impl Config {
                 with ConfigIterationFailure::TooLongUserConfigDirectory,
                 "{} since the resulting global config directory would be too long.", msg);
 
-        callback(global_config);
+        Ok(global_config)
+    }
+
+    fn iterate_over_config_files<F: FnMut(FilePath) -> CallbackProgression>(
+        mut callback: F,
+    ) -> Result<(), ConfigIterationFailure> {
+        let msg = "Unable to consider all possible config file paths";
+        let origin = "Config::iterate_over_config_files";
+
+        // prio 1: handle project local config file first
+        let local_project_config = Self::default_config_file_path();
+        if callback(local_project_config) == CallbackProgression::Stop {
```

Comment:
Does the callback print a debug message if it could not find the a config file and a warning if it finds one but cannot process the file?

Reply:
I added some trace messages informing the user when no config file was found or a config file was loaded.
If there was a problem loading a config file I print a warn message and continue to the next config file.